stackdriver/config/config_test.go

```diff
@@ -0,0 +1,82 @@
+package config
+
+import (
+	corev1 "k8s.io/api/core/v1"
+
+	"testing"
+)
+
+func TestNewStackdriverConfigFromConfigMap(t *testing.T) {
+	tests := []struct {
+		name           string
+		configMap      *corev1.ConfigMap
+		expectedConfig Config
+	}{
+		{
```

Comment:
nit: Coalesce to:
```
}{{
```
see for example: https://github.com/knative/serving/blob/master/pkg/reconciler/route/resources/names/names_test.go#L35